src/CallHandler.js

```diff
@@ -380,11 +381,17 @@ async function _startCallApp(roomId, type) {
             "Refusing to start conference call widget in " + roomId +
             " a conference call widget is already present",
         );
-        const ErrorDialog = sdk.getComponent("dialogs.ErrorDialog");
 
-        Modal.createTrackedDialog('Already have Jitsi Widget', '', ErrorDialog, {
-            title: _t('Call in Progress'),
-            description: _t('A call is already in progress!'),
+        Modal.createTrackedDialog('Already have Jitsi Widget', '', QuestionDialog, {
+            title: _t('End Call'),
+            description: _t('Remove the group call from the room?'),
+            button: _t('End Call'),
```

Comment:
I don't think this code path is ever guarded against not having permission to start a call, so the "end call" button could be shown to people who can't. The permission handling is done way at the end as part of adding the widget :(

Reply:
As yes, good point